static/src/javascripts/projects/common/modules/analytics/headlines-test-analytics.js

```diff
@@ -0,0 +1,40 @@
+define([
+    'common/utils/detect',
+    'common/utils/config',
+    'common/modules/analytics/beacon'
+], function (
+    detect,
+    config,
+    beacon
+) {
+
+    function clearHash() {
+        if (detect.hasPushStateSupport()) {
+            history.replaceState('', document.title, window.location.pathname);
+        } else {
+            window.location.hash = '';
+        }
+    }
+
+    return {
+        go: function () {
+            var hash = window.location.hash;
+
+            if (window.location.pathname === '/us') {
```

Comment:
You could do something like `page.contentType === 'Network Front' && page.edition === 'us'`.
I don't mind though,.